src/main/java/org/acra/ACRA.java

```diff
@@ -269,25 +269,20 @@ public static boolean isInitialised() {
      * @return true if the current process is the process running the SenderService.
      *          NB this assumes that your SenderService is configured to used the default ':acra' process.
      */
-    public static boolean isACRASenderServiceProcess(@NonNull Application app) {
-        final String processName = getCurrentProcessName(app);
+    public static boolean isACRASenderServiceProcess() {
+        final String processName = getCurrentProcessName();
         if (ACRA.DEV_LOGGING) log.d(LOG_TAG, "ACRA processName='" + processName + "'");
-        return (processName != null) && processName.equals(ACRA_PRIVATE_PROCESS_NAME);
+        return (processName != null) && processName.endsWith(ACRA_PRIVATE_PROCESS_NAME);
     }
 
     @Nullable
-    private static String getCurrentProcessName(@NonNull Application app) {
+    private static String getCurrentProcessName() {
         final int processId = android.os.Process.myPid();
-        final ActivityManager manager = (ActivityManager) app.getSystemService(Context.ACTIVITY_SERVICE);
-        List<ActivityManager.RunningAppProcessInfo> processInfos = manager.getRunningAppProcesses();
-        if (processInfos != null) {
-            for (final ActivityManager.RunningAppProcessInfo processInfo : processInfos) {
-                if (processInfo.pid == processId) {
-                    return processInfo.processName;
-                }
-            }
+        try {
+            return IOUtils.streamToString(new FileInputStream("/proc/"+processId+"/cmdline")).trim();
+        } catch (IOException e) {
+            return null;

```

Comment:
I've tested it with Xposed. No changes. As stated in the comments on the solution, this problem is outdated.